### DISCUSSION TITLE
Customisable site.webmanifest file to have mulitple Frigate Web Apps on Android (and iOS?).

### DISCUSSION THREAD

**Discussion by LaurenceGough** [blakeblackshear/frigate#12460] [Ideas]
Hi all,

Currently if someone is accessing Frigate externally via a proxy, cloud service etc and also locally for speed on their local network you can only add one Frigate as a web app (add to homescreen), which allows it to function as it's own app - and doing this it works awesomely.

Sadly if you want to add the other way of accessing your Frigate instance or I assume another Frigate instance it just shows as a webpage shortcut, opening a tab in your web browser instead. I believe this is due to the fact that the name "Frigate" in the site.webmanifest file is a duplicate.

I am no expert in how these work so I may be totally wrong.

A workaround is to use an app such as "Hermit" which allows you to convert any webpage into a web app, I am using this for now but it'd be nice to avoid.

I might look into creating the required changes for this to be customised, would anyone be interested?

Thanks

**Comment by NickM-27** [comment]
For separate apps I can see this being an issue though not sure how common multiple frigate instances are. 

at least personally I just use adguard to redirect the domain to the local instance while on the home network that way one app works via local while on the local network but remotely as well on the same web app

**Reply by LaurenceGough** [reply]
That's a great, handy way of doing it, unfortunately I use a cloud based adblocking DNS service which is always on. I don't believe there is anyway of accessing the hosts file on Android without root access either. :(
